db/post_migrate/20220302171443_rebake_old_avatar_service_urls.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class RebakeOldAvatarServiceUrls < ActiveRecord::Migration[6.1]
+  def up
+    execute <<~SQL
+      UPDATE posts SET baked_version = 0
+      WHERE cooked LIKE '%avatars.discourse.org%'
```

Comment:
this could lock up tables, it is a huge table scan, probably not on our servers, maybe on slow/huge discourse instances. mixed on shipping all this work out the the ecosystem if we can just wait a few months and catch it with other changes.

Reply:
We already ran this [on all our hosted sites](https://github.com/discourse-org/hosted-site/blob/main/db/post_migrate/20211125123708_rebake_avatar_urls.rb#L7) back in November and didn't notice any issues at all. But yeah I agree it could be more problematic on lower-powered servers.

So let's keep the site setting update, but defer the rebake for now?